charts/em-hrs-api/values.preview.template.yaml

```diff
@@ -47,8 +47,19 @@ java:
     CVP_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(CVP_BLOB_ACCOUNT_NAME);AccountKey=$(CVP_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
     VH_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(VH_BLOB_ACCOUNT_NAME);AccountKey=$(VH_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
     STORAGEACCOUNT_PRIMARY_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(HRS_BLOB_ACCOUNT_NAME);AccountKey=$(HRS_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
+    CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-ccd-data-store-api
+    CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-aat.service.core-compute-aat.internal
```

Comment:
I do not think hrs needs these vars, if ccd doe snot need them you can remove , you need to override this var/
    CCD_DATA_API_URL: http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal